src/styles/styles.scss

```
@@ -2574,3 +2598,582 @@ ol {
+      }
+
+      time,
+      p {
+        font-size: 14px;
+        line-height: 1.6;
+        margin-bottom: 0;
+      }
+
+      p {
+        margin-top: 10px;
+        // Two-line clamp keeps every card the same height.
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)